pkg/reconciler/delivery/delivery.go

```diff
@@ -39,7 +39,7 @@ const (
 	// AnnotationKey is the string used in ObjectMeta.Annotations map for any Route object
 	AnnotationKey = "KCDLastRouteUpdate"
 	// TimeFormat specifies the format used by time.Parse and time.Format
-	TimeFormat = time.ANSIC
+	TimeFormat = time.UnixDate
```

Comment:
Kubernetes usually uses time.RFC3339 as the format.

Reply:
This information is extremely helpful!!! I will definitely change it to RFC3339